diff --git a/dist/node/chunk-D3CUZ4fa.js b/dist/node/chunk-D3CUZ4fa.js
index fcc35e8af6f0178bd475a90d3e51efdb0d80d2c5..597541fef5731d45a1cf5c6a435b1cc38744471f 100644
--- a/dist/node/chunk-D3CUZ4fa.js
+++ b/dist/node/chunk-D3CUZ4fa.js
@@ -13,7 +13,7 @@ import { StringDecoder } from 'node:string_decoder';
 import require$$0$5, { resolve as resolve$1, parse as parse$4, basename, posix as posix$2, join as join$1 } from 'path';
 import require$$0$2, { inspect } from 'util';
 import { tmpdir } from 'os';
-import { normalizePath, loadConfigFromFile, createLogger, mergeConfig as mergeConfig$1, searchForWorkspaceRoot, build as build$1, transformWithEsbuild, createServer as createServer$1 } from 'vite';
+import { normalizePath, loadConfigFromFile, createLogger, mergeConfig as mergeConfig$1, searchForWorkspaceRoot, build as build$1, transformWithOxc, createServer as createServer$1 } from 'vite';
 import require$$0$4 from 'stream';
 import require$$2 from 'readline';
 import require$$3 from 'url';
@@ -45110,15 +45110,11 @@ async function createVitePressPlugin(siteConfig, ssr = false, pageToHashMap, cli
           if (isPageChunk(chunk)) {
             const hash = chunk.fileName.match(hashRE)[1];
             pageToHashMap[chunk.name.toLowerCase()] = hash;
-            bundle[name + "-lean"] = {
-              ...chunk,
+            this.emitFile({
+              type: "asset",
               fileName: chunk.fileName.replace(/\.js$/, ".lean.js"),
-              preliminaryFileName: chunk.preliminaryFileName.replace(
-                /\.js$/,
-                ".lean.js"
-              ),
-              code: chunk.code.replace(staticStripRE, `""`)
-            };
+              source: chunk.code.replace(staticStripRE, `""`)
+            });
             chunk.code = chunk.code.replace(staticRestoreRE, "");
           }
         }
@@ -49521,7 +49517,7 @@ async function renderHead(head) {
       const openTag = `<${tag}${renderAttrs(attrs)}>`;
       if (tag !== "link" && tag !== "meta") {
         if (tag === "script" && (attrs.type === undefined || attrs.type.includes("javascript"))) {
-          innerHTML = (await transformWithEsbuild(innerHTML, "inline-script.js", {
+          innerHTML = (await transformWithOxc(innerHTML, "inline-script.js", {
             minify: true
           })).code.trim();
         }
